workshops/dodge/README.md

```diff
@@ -298,16 +327,16 @@ This is our lose condition. Now we can punish the user for losing in this `gameO
 
 ### Game Over
 
-Even though it may feel like we've been writing all of our code in these two special p5js functions to make stuff in the canvas, we musn't forget that we are still just writing JavaScript. So let's define our own function, `gameOver()`, the way we would any other JS function. We'll place it underneath the `draw()` function.
+Even though it may feel like we've been writing all of our code in these two special p5.js functions to make stuff in the canvas, we musn't forget that we are still just writing JavaScript. So let's define our own function, `gameOver()`, the way we would any other JS function. We'll place it underneath the `draw()` function.
 
-```
+```js
 function gameOver() {
 }
 ```
 
-And let's display a Game Over screen.
+And let's display a game over screen.
```

Comment:
well then "game over"
but regardless, at least make the change throughout the document